From d6b1991f87e755678a523b9ed60e8f7e289267fc Mon Sep 17 00:00:00 2001
From: Alejandro Sierra <alejandro.sierra@nxp.com>
Date: Mon, 26 Sep 2016 16:37:19 -0500
Subject: [PATCH 1255/5242] MLK-13287 ARM: dts: imx: support for epop 6sxscm
 evb board

commit  2da8ef328880ff01c1e579bd567da4f2c7e14431 from
https://source.codeaurora.org/external/imx/linux-imx.git

Add support for the epop i.MX6SX SCM EVB board
The epop variant contains an eMMC(512MB) within the
POP package.

Support the next features for epop 6sxscm EVB:

 - Regular epop board
 - M4 support

Signed-off-by: Alejandro Sierra <alejandro.sierra@nxp.com>
Signed-off-by: Juan Gutierrez <juan.gutierrez@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/Makefile                      |    4 +++-
 arch/arm/boot/dts/imx6sxscm-epop-evb-ldo.dts    |    9 +++++++++
 arch/arm/boot/dts/imx6sxscm-epop-evb-m4-ldo.dts |    9 +++++++++
 3 files changed, 21 insertions(+), 1 deletion(-)
 create mode 100644 arch/arm/boot/dts/imx6sxscm-epop-evb-ldo.dts
 create mode 100644 arch/arm/boot/dts/imx6sxscm-epop-evb-m4-ldo.dts

diff --git a/arch/arm/boot/dts/Makefile b/arch/arm/boot/dts/Makefile
index b34d003..2dd9316 100644
--- a/arch/arm/boot/dts/Makefile
+++ b/arch/arm/boot/dts/Makefile
@@ -587,7 +587,9 @@ dtb-$(CONFIG_SOC_IMX6SX) += \
 	imx6sxscm-1gb-evb-m4-ldo.dtb \
 	imx6sxscm-1gb-evb-mqs-ldo.dtb \
 	imx6sxscm-1gb-evb-sai-ldo.dtb \
-	imx6sxscm-1gb-evb-btwifi-ldo.dtb
+	imx6sxscm-1gb-evb-btwifi-ldo.dtb \
+	imx6sxscm-epop-evb-ldo.dtb \
+	imx6sxscm-epop-evb-m4-ldo.dtb
 dtb-$(CONFIG_SOC_IMX6UL) += \
 	imx6ul-14x14-evk.dtb \
 	imx6ul-14x14-evk-btwifi.dtb \
diff --git a/arch/arm/boot/dts/imx6sxscm-epop-evb-ldo.dts b/arch/arm/boot/dts/imx6sxscm-epop-evb-ldo.dts
new file mode 100644
index 0000000..62bc1ff
--- /dev/null
+++ b/arch/arm/boot/dts/imx6sxscm-epop-evb-ldo.dts
@@ -0,0 +1,9 @@
+/*
+ * Copyright (C) 2016 Freescale Semiconductor, Inc.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2 as
+ * published by the Free Software Foundation.
+ */
+#include "imx6sxscm-512mb-evb-ldo.dts"
+#include "imx6sxscm-emmc.dtsi"
diff --git a/arch/arm/boot/dts/imx6sxscm-epop-evb-m4-ldo.dts b/arch/arm/boot/dts/imx6sxscm-epop-evb-m4-ldo.dts
new file mode 100644
index 0000000..ab60602
--- /dev/null
+++ b/arch/arm/boot/dts/imx6sxscm-epop-evb-m4-ldo.dts
@@ -0,0 +1,9 @@
+/*
+ * Copyright (C) 2016 Freescale Semiconductor, Inc.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2 as
+ * published by the Free Software Foundation.
+ */
+#include "imx6sxscm-512mb-evb-m4-ldo.dts"
+#include "imx6sxscm-emmc.dtsi"
-- 
1.7.9.5

